modules/default/calendar/vendor/ical.js/node-ical.js

```diff
@@ -26,6 +26,17 @@ exports.parseFile = function(filename){
 
 var rrule = require('rrule').RRule
 
+function getLocaleISOString(date) {
+	var year = date.getFullYear().toString(10).padStart(4,'0');
+	var month = date.getMonth().toString(10).padStart(2,'0');
```

Comment:
getMonth() returns a month between 0 and 11, not between 1 and 12 so this creates an invalid date.

Reply:
thanks for the catch.  I just put up PR #1914 to address this.